component/org.wso2.carbon.identity.casque.authenticator/src/main/java/org/wso2/carbon/identity/casque/authenticator/authenticator/CasqueConfig.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.casque.authenticator.authenticator;
+
+import org.wso2.carbon.identity.casque.authenticator.constants.CasqueAuthenticatorConstants;
+import org.wso2.carbon.identity.casque.authenticator.exception.CasqueException;
+import org.wso2.carbon.utils.CarbonUtils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.IOException;
+import java.net.InetAddress;
+
+/**
+ * CASQUE SNR Authenticator Configurations
+ */
+public class CasqueConfig {
+
+    public static byte[] radiusSecret = null;
+    public static InetAddress casqueAddress = null;
+    public static int casquePort = 0;
+    public static int localPort = 0;
+    private static boolean configLoaded = false;
+
+    /**
+     * Configuration line readings
+     * @param line ,
+     * @throws IOException
+     */
```

Comment:
In method level or class level comments add a meaningful description to the parameters. Check this throughout the code